Backstage Glossary Documentation (#4073)
backstage/backstage: docs/glossary.md

---
id: Glossary
title: Backstage Glossary
# prettier-ignore
description: List of all abbreviations, terms, phrases used in Backstage together with its explanations.
---

> # Backstage Glossary

Reviewer: In practice, this will create a duplicate header, with line 3 displaying at the top and this right under it. Suggest this should be either Overview, or my suggestion, removed entirely?

```suggestion
```

---
id: Glossary
title: Backstage Glossary

> description:
>   List of all abbreviations, terms, phrases used in Backstage together with its
>   explainations.

Reviewer 2: Annoying SEO hack for Docusaurus. Ref: https://github.com/backstage/backstage/pull/3927

```suggestion
# prettier-ignore
description: List of all abbreviations, terms, phrases used in Backstage together with its explanations.
```

---

# Backstage Glossary

The Backstage Glossary lists all abbreviations, terms or phrases that are used
in Backstage together with its explanation. We highly recommend you to use these
to contribute to a consistent Backstage terminology.

### Backstage User Profiles

There are three main user profiles for Backstage: the integrator, the
contributor, and the software engineer.

| Term              | Explanation                                                                                              |
| ----------------- | -------------------------------------------------------------------------------------------------------- |
| Integrator        | The **integrator** hosts the Backstage app and configures which plugins are available to use in the app. |
| Contributor       | The **contributor** adds functionality to the app by writing plugins.                                    |

> | Software Engineer | The **software engineer** uses the app's functionality and interacts with its plugins.                   |

Reviewer 2: Should "Software Engineer" be just "User"?

All kinds of folks use Backstage no, Software, Hardware, Data, non-Engineer :P

Author: The discussion yesterday lead to using these. I'm all up for replacing Software Engineer with User to include other type of users that might *use* Backstage as well such as Designers, Data Scientists, Managers etc. 

@backstage/maintainers what do you think?

Reviewer: I'm aligned with both of your points here. I know as I've pitched Backstage internally to those who would foot the bill and provide velocity, my teams have asked me questions like, "what is in it for the product owners?" "why would the scrum masters use this?" "why would an engineering manager ever want to use Backstage?"

So perhaps even if "Software Engineer" is left as the third persona, the description can make it more clear that there is expected to be a much larger audience (at least, I expect that) covering that persona/user profile, such as designers, data scientists, architects,.

```suggestion
| Software Engineer | The **software engineer** uses the app's functionality and interacts with its plugins. In practice, this profile covers the various roles that help deliver software, from the Software Engineer themselves, to Designers, Data Scientists, Product Owners, Engineering Managers, etc. |
```
(Note this suggestion will fail `prettier` 😉 )
